assets/sparksql/Dockerfile

```
@@ -0,0 +1,13 @@
+# FROM spark:python3
+
+FROM spark:4.0.0
+
+USER root
+
+RUN apt-get update;
+
+RUN apt-get install net-tools netcat;
+
+RUN pip3 install pyspark;
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)